src/JsonNormalizer.php

```diff
@@ -1,36 +1,45 @@
 <?php
 
-
 namespace Tuf;
 
+/**
+ * Provdes normalization to convert an array to a canonical JSON string.
+ */
 class JsonNormalizer
 {
     /**
-     * Computes the canonical JSON string representation of an assoc. array.
+     * Encodes an associative array into a string of canonical JSON.
+     *
+     * @param mixed[] $structure
+     *     The associative array of JSON data.
+     *
+     * @return string
+     *     An encoded string of normalized, canonical JSON data.
      *
      * @todo This is a very incomplete implementation of
      *     http://wiki.laptop.org/go/Canonical_JSON.
      *     Consider creating a separate library under php-tuf just for this?
-     *
-     * @param array $structure
-     *
-     * @return string
      */
-    public static function asNormalizedJson($structure)
+    public static function asNormalizedJson(array $structure) : string
     {
-        if (!is_array($structure)) {
-            throw new \Exception("Array of keys to canonicalize is required");
-        }
-
```

Comment:
Note that the exception is not necessary if we add the typehint to the method signature.